File: app/src/main/java/com/musicseque/fragments/UploadFragment.java
package com.musicseque.fragments;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import android.widget.LinearLayout;

import com.musicseque.R;

public class UploadFragment extends Fragment implements View.OnClickListener {
    View view;
    LinearLayout llPhotos, llAudio, llVideos;
    ImageView ivUploadVideo, ivUploadAudio, ivUploadPhoto;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        view = inflater.inflate(R.layout.fragment_upload, container, false);
        initOtherViews();
        initViews();
        listeners();
        return view;
    }

    private void initOtherViews() {

    }

    private void initViews() {
        llPhotos = (LinearLayout) view.findViewById(R.id.llPhotos);
        llAudio = (LinearLayout) view.findViewById(R.id.llAudio);
        llVideos = (LinearLayout) view.findViewById(R.id.llVideos);
        ivUploadVideo = (ImageView) view.findViewById(R.id.ivUploadVideo);
        ivUploadAudio = (ImageView) view.findViewById(R.id.ivUploadAudio);
        ivUploadPhoto = (ImageView) view.findViewById(R.id.ivUploadPhoto);

    }

    private void listeners() {
        llPhotos.setOnClickListener(this);
        llAudio.setOnClickListener(this);
        llVideos.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId())
        {
            case R.id.llPhotos:

                break;
            case R.id.llAudio:

                break;
            case R.id.llVideos:

                break;
        }
    }
}

File: app/src/main/java/com/musicseque/fragments/UploadFragment.kt
package com.musicseque.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView
import android.widget.LinearLayout
import com.musicseque.R

class UploadFragment : Fragment(), View.OnClickListener {
    var v: View? = null
    var llPhotos: LinearLayout? = null
    var llAudio: LinearLayout? = null
    var llVideos: LinearLayout? = null
    var ivUploadVideo: ImageView? = null
    var ivUploadAudio: ImageView? = null
    var ivUploadPhoto: ImageView? = null
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        v = inflater.inflate(R.layout.fragment_upload, container, false)
        initOtherViews()
        initViews()
        listeners()
        return v
    }

    private fun initOtherViews() {}
    private fun initViews() {
        llPhotos = view!!.findViewById<View>(R.id.llPhotos) as LinearLayout
        llAudio = view!!.findViewById<View>(R.id.llAudio) as LinearLayout
        llVideos = view!!.findViewById<View>(R.id.llVideos) as LinearLayout
        ivUploadVideo = view!!.findViewById<View>(R.id.ivUploadVideo) as ImageView
        ivUploadAudio = view!!.findViewById<View>(R.id.ivUploadAudio) as ImageView
        ivUploadPhoto = view!!.findViewById<View>(R.id.ivUploadPhoto) as ImageView
    }

    private fun listeners() {
        llPhotos!!.setOnClickListener(this)
        llAudio!!.setOnClickListener(this)
        llVideos!!.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.llPhotos -> {
            }
            R.id.llAudio -> {
            }
            R.id.llVideos -> {
            }
        }
    }
}